src/pages/HomePage.tsx

```diff
@@ -1,5 +1,154 @@
+import { HiFire } from 'react-icons/hi';
+import { MdOutlineSearch } from 'react-icons/md';
+import Article from '@components/Article';
+import BottomNavigation from '@components/BottomNavigation';
+import HeaderText from '@components/HeaderText';
+
+const CHARACTER_SRC = '/img/character.png';
+
 const HomePage = () => {
-  return <h2>홈페이지입니다</h2>;
+  return (
+    <div className="relative flex justify-center w-full overflow-y-scroll">
+      <div className="w-full max-w-md flex flex-col gap-36 shrink-0">
```

Comment:
p4: 홈페이지같은 경우는 전체 페이지가 스크롤 되어야 하는 것 같습니다. 지금은 최신 이야기 부분의 하단이 잘리는 거 같은데 최상단 태그에 패딩을 주는게 어떨까요?
(갑자기 든 생각인데 스크롤 될 때만 BottomNavigation이 보이게 하는 기능도 있으면 좋겠네요... ㅋㅋㅋ)
```suggestion
    <div className="relative flex justify-center w-full overflow-y-scroll pb-[5rem]">
      <div className="w-full max-w-md flex flex-col gap-36 shrink-0">
```

Reply:
p5: HTML 태그명도 시멘틱하게 작성해주시면 더 좋을 것 같습니다~!